backend/app/services/internal/auth_service.py

```
@@ -0,0 +1,118 @@
+
+    def update_password(self, email: str, old_password: str, new_password: str) -> dict:
+        # """Update user's password."""
+        user = self.user_repo.get_user_by_email(email, includePassword=True)
+        if not user:
+            raise NotFoundException(message="User not found")
+
+        if not verify_password(old_password, user.password_hash):
+            raise UnauthorizedException(message="Invalid old password")
+        
+        new_password_hash = get_password_hash(new_password)
+        updated_user = self.user_repo.update_password(user.id, new_password_hash)
```

Ruff (0.14.10)

Local variable `updated_user` is assigned to but never used

Remove assignment to unused variable `updated_user`

(F841)